tests/integration/query/commits/simple_test.go

```diff
@@ -315,3 +313,118 @@ func TestQueryCommitsWithFieldIDFieldWithUpdate(t *testing.T) {
 
 	testUtils.ExecuteTestCase(t, test)
 }
+
+func TestQuery_CommitsWithAllFieldsWithUpdate_NoError(t *testing.T) {
+	test := testUtils.TestCase{
+		Actions: []any{
+			updateUserCollectionSchema(),
+			testUtils.CreateDoc{
+				Doc: `{
+						"name":	"John",
+						"age":	21
+					}`,
+			},
+			testUtils.UpdateDoc{
+				Doc: `{
+					"age":	22
+				}`,
+			},
+			testUtils.Request{
+				Request: `
+					query {
+						commits {
+							cid
+							collectionID
+							delta
+							docID
+							fieldId
+							fieldName
+							height
+							links {
+								cid
+								name
+							}
+						}
+					}
+				`,
+				Results: []map[string]any{
+					{
+						"cid":          "bafybeiddpjl27ulw2yo4ohup6gr2wob3pwagqw2rbeaxxodv4ljelnu7ve",
+						"collectionID": int64(1),
+						"delta":        []uint8([]byte{0x16}),
```

Comment:
question: should this be a base64 encoding instead?

Reply:
yes. Trying to figure something out for this. It si base64 when using CLI and HTTP clients but not when using the go client.

Reply:
So it's base64 with CLI and HTTP clients because that's how `json.Marshal` will convert `[]uint8` to `string` representation. I've added support for `[]uint8` result assertion when using the CLI and HTTP clients.